plivo/utils/signature_v3.py

```diff
@@ -0,0 +1,121 @@
+import sys
+from hmac import new as sign
+from hashlib import sha256
+from .validators import *
+
+PY2 = True if sys.version_info.major == 2 else False
+
+if PY2:
+    from urlparse import urlparse, urlunparse, parse_qs
+    from base64 import encodestring as encode
+else:
+    from urllib.parse import urlparse, urlunparse, parse_qs
+    from base64 import encodebytes as encode
+
+
+
+def string_format(value):
+    if isinstance(value, bytes):
+        return ''.join(chr(x) for x in bytearray(value))
+    if isinstance(value, (int, float, bool)):
+        return str(value)
+    if isinstance(value, list):
+        return [string_format(x) for x in value]
+    if PY2 == True and isinstance(value, unicode):
+        return value.decode('utf-8')
+    return value
+
+
+def get_map_from_query(query):
+    res_map = dict()
+    for key, value in parse_qs(query).items():
+        res_map[string_format(key)] = string_format(value)
+    return res_map
+
+
+def get_sorted_query_string(params):
+    keys = sorted(params.keys())
+    res_params = []
+    for key in keys:
+        value = params[key]
+        if isinstance(value, list):
+            res_params.append(
+                '&'.join(['{}={}'.format(string_format(key), val) for val in sorted(string_format(value))]))
```

Comment:
Please remove = between key value pairs.
In spec urls are generated with key value pairs appended without "="
please refer example
https://abcd.com/xyz/:8080?a=p.CallUuidfsfs-dfsdfs-fsfs-fsdf-sssCaller+14158675310Digits1234From1234From+14158675310To+18005551212